operator/pkg/model/translation/envoy_cluster.go

```diff
@@ -29,101 +31,74 @@ const (
 	HTTPVersion3          HTTPVersionType = 3
 )
 
-type ClusterMutator func(*envoy_config_cluster_v3.Cluster) *envoy_config_cluster_v3.Cluster
-
-// WithClusterLbPolicy sets the cluster's load balancing policy.
-// https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/upstream/load_balancing/load_balancers
-func WithClusterLbPolicy(lbPolicy int32) ClusterMutator {
-	return func(cluster *envoy_config_cluster_v3.Cluster) *envoy_config_cluster_v3.Cluster {
-		if cluster == nil {
-			return cluster
-		}
-		cluster.LbPolicy = envoy_config_cluster_v3.Cluster_LbPolicy(lbPolicy)
-		return cluster
+func (i *cecTranslator) clusterMutators(grpcService bool, appProtocol string) []ClusterMutator {
+	res := []ClusterMutator{
+		withConnectionTimeout(5),
+		withIdleTimeout(i.Config.ClusterConfig.IdleTimeoutSeconds),
+		withClusterLbPolicy(int32(envoy_config_cluster_v3.Cluster_ROUND_ROBIN)),
+		withOutlierDetection(true),
```

Comment:
It seems like some of these values were not set for HTTP clusters previously? Is this a change in behavior, or were we setting them somewhere else?

Reply:
previously it was set in cec_translator as per below, the same value is used.

https://github.com/cilium/cilium/blob/5dca7addd707a00e571db2af1c024108ca09d6e2/operator/pkg/model/translation/cec_translator.go#L349